test/steps/http.go

```diff
@@ -56,6 +59,7 @@ func (data *Data) httpPostRequestOnServiceWithPathAndBody(serviceName, path stri
 }
 
 func (data *Data) httpPostRequestOnServiceIsSuccessfulWithinMinutesWithPathAndBody(serviceName string, timeoutInMin int, path string, body *messages.PickleStepArgument_PickleDocString) error {
+	path = data.ResolveWithScenarioContext(path)
 	framework.GetLogger(data.Namespace).Infof("httpPostRequestOnServiceWithPathAndBody with service %s, path %s, %s bodyContent %s and timeout %d", serviceName, path, body.GetMediaType(), body.GetContent(), timeoutInMin)
```

Comment:
These loggers should be in debug instead?

Reply:
yes

Reply:
changed